packages/@aws-cdk/aws-ecs/lib/container-definition.ts

```diff
@@ -516,6 +531,23 @@ export class ContainerDefinition extends CoreConstruct {
     }));
   }
 
+  /**
+   * This method adds one or more resources to the container.
+   */
+  public addInferenceAcceleratorResource(...inferenceAcceleratorResources: string[]) {
+    if (!this.taskDefinition.inferenceAccelerators) {
+      throw new Error('InferenceAccelerator resource(s) defined in container definition without specifying any inference accelerators in task definition.');
```

Comment:
Maybe we don't need this error since ultimately this doesn't work because of https://github.com/aws/aws-cdk/pull/13950/files#diff-3654071c3c543c7a4938c726b62afa9b5ad382c05d4d3adff32f883146ff7148R547